pkg/types/openstack/validation/platform.go

```diff
@@ -64,14 +67,29 @@ func ValidatePlatform(p *openstack.Platform, fldPath *field.Path, fetcher ValidV
 	// Validate VIP Values
 	if err := validate.IP(p.APIVIP); err != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("apiVIP"), p.APIVIP, err.Error()))
-	}
-
-	if err := validate.IP(p.IngressVIP); err != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("ingressVIP"), p.IngressVIP, err.Error()))
+	} else if n != nil {
+		expectedAPIVIP, _ := cidr.Host(&n.MachineCIDR.IPNet, 5)
```

Comment:
hmm, the user is not alllowed to set the IP?? Why do we even have this field in the install-config then???

Reply:
oops, the comment is mean for change on the next line...

Reply:
That's a good point, yeah. I think trown (who wrote this originally IIRC) thought that's the only way to pass the values throughout the installer, but we can actually just use functions that return the VIPs we need throughout.

Going to fix that.

Reply:
The top commit removes the VIPs from install-config:

https://github.com/openshift/installer/pull/1959/commits/6659b86d4b62f6f30f3d227f2b118e3135063e64